lang/algol68g/files/patch-includedir.diff

```diff
@@ -0,0 +1,24 @@
+--- configure.orig	2022-03-13 04:55:26.000000000 +0800
++++ configure	2022-05-21 21:25:34.000000000 +0800
```

Comment:
Since you're using `use_autoreconf yes`, patch configure.ac, not configure, for this change to have any effect. Or possibly the newly generated configure script already contains this change.